modules/amp-react/pages/6-share.adoc

```diff
@@ -0,0 +1,111 @@
+= Social Sharing
+
+This tutorial shows how to integrate social sharing using AMP social buttons.
+
+== Getting started
+Social sharing is enabled by default in AMP player's UI for further usage. 
+
+If social sharing is not required it can be turned off through player config. See https://docs.player.akamai.com/static/amp-web-sdk/amp-web-react/akamai.amp.react.Share.html[Share Configuration] for more information
+
+[source,javascript]
+----
+var config = {
+    plugins: {
+        react: {
+            share: {
+                enabled: false,
+            }
+        }
+    }
+}
+----
+Or through CSS adding the following rule on page styling.
+
+[source,css]
+----
+.amp-share {
+    display: none;
+}
+----
+
+== How to use built-in social sharing UI
+
+Player offers a built-in share panel and buttons along with callbacks which can be connected to customer's favorite social provider.
+
+Player offers `share` event callback to hook users choice, within in payload can be use to perform an action such like prompting a redirect to the social network url or show a custom overlay.  
```

Comment:
@mchacong 
`within in the payload` sounds duplicated, maybe `within the payload` is better.

Reply:
fixed